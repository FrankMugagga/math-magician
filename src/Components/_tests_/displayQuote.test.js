import React from 'react';
// eslint-disable-next-line import/no-extraneous-dependencies
import renderer from 'react-test-renderer';
import DisplayQuote from '../displayQuote';

describe('DisplayQuotes renders correctly', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<DisplayQuote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
