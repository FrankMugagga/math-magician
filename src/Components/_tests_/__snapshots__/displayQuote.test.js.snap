// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DisplayQuotes renders correctly renders correctly 1`] = `
<div>
  <ul
    className="quote"
  />
</div>
`;
